libs/utils/src/lib/logging.ts

```diff
@@ -0,0 +1,14 @@
+import { createLogger, format, transports } from 'winston';
+
+const { timestamp, json, colorize } = format;
+
+// TODO: cust class appending appending more info per log
+export const getLogger = (source: string, tag: string) => {
+  const logger = createLogger({
+    level: 'info',
+    format: format.combine(timestamp(), colorize(), json()),
```

Comment:
It appears that the ANSI color codes are not rendering correctly, which hinder readability. The raw ANSI codes are shwon: 

<img width="1202" alt="Screenshot 2023-05-09 at 12 38 24 PM" src="https://user-images.githubusercontent.com/12991700/237058238-a9f1ab91-700d-4e03-a4d7-ef30a6ed5b11.png">

Reply:
that's because it's being output as JSON for the benfit of datadog. for now, I'm just going to drop colorization.